### ISSUE TITLE
WebUI crashing on Colab

### ISSUE THREAD

**Issue by inconnu26** [WASasquatch/StableDiffusionUI-TomRiddle#1]
Here is the error message I see on Colab for this projet:

AttributeError                            Traceback (most recent call last)
[<ipython-input-1-32b87f537004>](https://localhost:8080/#) in <module>
     14         txt2img=webui.wrap_gradio_gpu_call(modules.txt2img.txt2img),
     15         img2img=webui.wrap_gradio_gpu_call(modules.img2img.img2img),
---> 16         run_extras=webui.wrap_gradio_gpu_call(webui.run_extras),
     17         run_pnginfo=webui.run_pnginfo
     18 )

AttributeError: module 'webui' has no attribute 'run_extras'

**Comment by ShinaZin** [comment]
As a workaround you can replace 16-17 lines like this:
```python
run_extras=webui.wrap_gradio_gpu_call(modules.extras.run_extras),
run_pnginfo=modules.extras.run_pnginfo
```

**Comment by WASasquatch** [comment]
Yeah it looks like there has been changed with WebUI.  I only set this up as a auto login for someone and cleaned up install method. I'll see if I can patch that when I get some time. May have to use Voldemort or another version of you can't patch
